### DISCUSSION TITLE
Suggest someone for the Environment Variables podcast

### DISCUSSION THREAD

**Discussion by mrchrisadams** [Green-Software-Foundation/discussions#31] [Environment Variables]
Hey folks, 

I want to try a thing related to the podcast I co-host [Environment Variables](https://podcast.greensoftware.foundation/), accessible at https://podcast.greensoftware.foundation/

If there is someone you'd like to hear from in the podcast talking about their work, you can suggest them with the form below:

https://docs.google.com/forms/d/e/1FAIpQLSfBSJCeFzQTmo5TOXbIDGWH5DJLYaAap7lecRIoUXRqj6a0bA/viewform

The form is an veeery early version, and I'll working on a way to accept submissions if you can't access google docs where you work, but it should at least allow you to nominate someone.

Give it a go!

**Comment by greenhsu123** [comment]
Unfortunately, google form doesn't work for some of us :( But I recommend [Holly Cummins ](https://hollycummins.com)

**Comment by NAMRATA-WOKE** [comment]
@mrchrisadams Simply because he just got promoted to this role, and because of how vocal Salesforce has been recently in support of green software principles and practice. https://www.linkedin.com/in/bgamazay/

**Comment by Celineleopoul** [comment]
@mrchrisadams - We launched our data portal with free granular electricity data. We think it will be great to have @ciril-emaps in the podcast to discuss the opportunities that this portal provides.

**Comment by WBurggraaf** [comment]
Is there a platform where I can listen to this podcast? Thanks!

**Reply by mrchrisadams** [reply]
hi @WBurggraaf I've updated the post with a link to Environment Variables, the podcast I was talking about - I should have mentioned that in the initial post, and not assumed everyone already knew what I was talking about 😅
